freqtrade/exchange/__init__.py

```diff
@@ -294,6 +294,11 @@ def get_ticker_history(pair: str, tick_interval: str, since_ms: Optional[int] =
         while not since_ms or since_ms < till_time_ms:
             data_part = _API.fetch_ohlcv(pair, timeframe=tick_interval, since=since_ms)
 
+            # Because some exchange sort Tickers ASC and other DESC.
```

Comment:
It is not necessary. CCXT library does sorting for you. Here is what the [documentation](https://github.com/ccxt/ccxt/wiki/Manual) says:
```
The list of candles is returned sorted in ascending (historical) order, oldest candle first, most recent candle last.
```

Reply:
@glonlas how about dropping an issue to CCXT github about this? Not that we really care, we can of course sort those but still, it would be better fixed on the source.